src/overlays/actors/ovl_End_Title/z_end_title.c

```diff
@@ -27,13 +27,134 @@ const ActorInit End_Title_InitVars = {
     (ActorFunc)EndTitle_Update,
     (ActorFunc)EndTitle_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_End_Title/EndTitle_Init.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_End_Title/EndTitle_Destroy.s")
+#include "z_end_title_gfx.c"
+
+void EndTitle_Init(Actor* thisx, GlobalContext* globalCtx) {
+    EndTitle* this = THIS;
+
+    this->theEndTitleCardAlpha = 0;
+    this->zeldaTitleCardAlpha = 0;
+    this->ocarinaTitleCardAlpha = 0;
```

Comment:
imo given the context of the overlay i dont think its neccesary to specify that they are title cards in the variable name.
endAlpha, tlozAlpha, and ootAlpha are what I would go with personally